bigflow/dataflow.py

```diff
@@ -18,8 +18,9 @@ def __init__(
             pipeline_options: PipelineOptions = None,
             entry_point_arguments: typing.Optional[dict] = None,
             wait_until_finish: bool = True,
-            execution_timeout: int = 3600000,
-            test_pipeline: Pipeline = None
+            execution_timeout: int = None,
+            test_pipeline: Pipeline = None,
+            job_execution_timeout: int = DEFAULT_JOB_EXECUTION_TIMEOUT
```

Comment:
Without any context, a user has no clue why there are 2 execution timeouts. I suggest using a single parameter - `execution_timeout` and an additional parameter, like `pipeline_level_execution_timeout_shift`, so the final execution timeout will be `execution_timeout - pipeline_level_execution_timeout_shift`.

@anjensan what do you think?

Reply:
+1 for having `execution_timeout` as main entry for users.
IMO `pipeline_level_execution_timeout_shift` is not really needed to be exposed to user (but we might still have such option as attribute of 'Job' objects).

Reply:
Ok, will change